meson.build

```
@@ -415,6 +415,20 @@ if build_system_helper
   cdata.set('USE_SYSTEM_HELPER', '1')
 endif
 
+foreach function : [
+  'close_range',
+]
+  macro = 'HAVE_' + function.underscorify().to_upper()
+  config.set(macro, cc.has_function(function) ? 1 : false)
```

should be

```suggestion
  cdata.set(macro, cc.has_function(function) ? 1 : false)
```

(and maybe also what @TingPing suggested, I need to test that on an older/newer OS that does/doesn't have the function)